extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -21,21 +21,31 @@ import {removeFragment} from '../../../src/url';
 
 class AmpAccordion extends AMP.BaseElement {
 
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {?NodeList} */
+    this.sections_ = null;
+
+    /** @private {?string} */
+    this.id_ = null;
```

Comment:
nvm, you are doing assertString on it, probably better to leave as nullable then